spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -751,7 +754,9 @@ def update_environment_menu(self):
                 action,
                 menu=self.console_environment_menu
             )
+
         self.console_environment_menu._render()
+        self.console_environment_menu._set_icons()
```

Comment:
Should this too calls be available from the Spyder menu definition as a public accesible method like `SpyderMenu.render_with_icons` or `SpyderMenu.render_menu`